### DISCUSSION TITLE
Variable list collapsable

### DISCUSSION THREAD

**Discussion by softwareklinic** [grafana/grafana#40595] [Dashboards, Panels & Visualizations]
We usually end up with a big list of variables since we have so many data elements that we use to make the dashboard very dynamic - what I feel is if we can make the Variables section collapsable just like the Dashboard row - it would be great - we can select the variable values and then collapse

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
